src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -119,4 +119,11 @@ private static function isExpired(string $date): bool
 
         return false !== $date && new \DateTime() > $date->modify('last day of this month 23:59:59');
     }
+
+    private static function daysBeforeExpiration(string $date): string
+    {
+        $date = \DateTime::createFromFormat('d/m/Y', '01/'.$date);
+
+        return (new \DateTime())->diff($date->modify('last day of this month 23:59:59'))->format('in %R%a days');
```

Comment:
does the `in` work fine if you run this paste the EOL date ?

Reply:
if the date is passed, it is the first `if` that takes precedence which will write `expired`
or i do not understand what do you mean @stof

Reply:
I missed the fact that this was used only for future dates